.github/workflows/ci.yml

```
@@ -88,40 +92,69 @@ jobs:
         docker rmi $(docker image ls -aq)
         df -h
 
-    - name: Deploy chart
+    - name: Deploy nginx chart ☕️
       run: |
-        helm dependency update ${CHART}
-        helm install \
-          --set rasax.disableTelemetry=true \
-          --set duckling.enabled=false \
-          --set rasax.initialUser.password=${INITIAL_USER_PASSWORD} \
-          --set nginx.service.type=ClusterIP \
-          --set ingress.hosts[0].host=,ingress.hosts[0].paths={/} \
-          ${RELEASE_NAME} ${CHART}
-
-    - name: Wait for deployment to be ready
+        helmfile -f .github/deployments/nginx-helmfile.yaml sync
```

```suggestion
        helmfile -f .github/deployments/ingress-nginx-helmfile.yaml sync
```
You have to change the file name as well :)